module/Application/src/Model/Service/Analytics/GoogleAnalyticsService.php

```
@@ -0,0 +1,60 @@
+<?php
+
+namespace Application\Model\Service\Analytics;
+
+use Application\Model\Service\AbstractService;
+use Opg\Lpa\Logger\LoggerTrait;
+use TheIconic\Tracking\GoogleAnalytics\Analytics;
+
+class GoogleAnalyticsService extends AbstractService
+{
+    use LoggerTrait;
+
+    /**
+     * Call Google Analytics to register a page view for the PDFs
+     *
+     * @param $pagePath Page url, excluding the domain e.g. /general/mypage
+     * @param $pageTitle Title of the page
+     */
+    function sendPageView($pagePath, $pageTitle)
+    {
+        $this->getLogger()->debug('Sending analytics page view ' . $pagePath . ' , ' . $pageTitle);
+
+        $clientId = $this->getAnalyticsClientId();
+
+        $this->getLogger()->debug('Client ID: ' . $clientId);
+
+        $analytics = new Analytics(true);
+
+        $analytics->setProtocolVersion('1')
+            ->setTrackingId('UA-33184303-1')
+            ->setClientId($clientId)
+            ->setDocumentPath($pagePath)
+            ->setDocumentTitle($pageTitle)
+            ->setAnonymizeIp(true);
+
+        $response = $analytics->sendPageview();
+
+        $this->getLogger()->debug('Analytics response code: ' . $response->getHttpStatusCode());
+    }
+
+    /**
+     * Get the client ID from the cookie
+     *
+     * @return clientId
+     */
+    function getAnalyticsClientId()
+    {
+        if (isset($_COOKIE['_ga'])) {
+            list($version, $domainDepth, $cid1, $cid2) = explode('.', $_COOKIE['_ga'], 4);
+            $contents = array('version' => $version, 'domainDepth' => $domainDepth, 'cid' => $cid1 . '.' . $cid2);
+            $cid = $contents['cid'];
+        } else {
+            $cid = gaGenerateUUID();
```

Not sure what this function is. Should we just throw an exception here so the page hit doesn't get recorded?